live-examples/css-examples/backgrounds-and-borders/border-bottom-left-radius.html

```diff
@@ -0,0 +1,41 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="border-bottom-left-radius">
+
+    <div class="example-choice" initial-choice="true">
+        <pre><code id="example_one" class="language-css">border-bottom-left-radius: 80px 80px;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_two" class="language-css">border-bottom-left-radius: 250px 100px;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_three" class="language-css">border-bottom-left-radius: 50%;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_four" class="language-css">border-bottom-left-radius: 50%;
+border: black 10px double;
+background-clip: content-box;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+</section>
+
+<div id="output" class="output large hidden">
+
+    <section id="default-example" class="default-example">
+        <div id="example-element" class="transition-all">This is a box with bottom left rounded corner.</div>
```

Comment:
Grammar nit: I think this should be "This is a box with a bottom left rounded corner." - here and in the other files.

Reply:
Got it!